De novo repeat annotation documnet.tex

```diff
@@ -1,321 +1,508 @@
+	java ClassifyConsensusSequences
+	\end{lstlisting}
+	This should generate 5 output files: known.txt, partial.txt, check.txt, notKnown.fa, notknown.fa.gff. 
+	Then we need to further annotate these notKnown.fa consensus sequences. 
+	
+	\pagebreak
+	
+	\subsubsection{Filter sequences.}
+	This step contains three parts: 1) Identify potential protein sequences; 2) Identify GB\_TE sequences; 3) Identify retrovirus sequences. You can run each part separately in parallel to save time. From these three steps, you will get three output files for following steps: 1) notKnown.fa.spwb.gff, 2) notKnown.fa.tewb.gff, 3) notKnown.fa.ervwb.gff. \\
+	
+	\noindent First download uniprot protein dataset from \url{ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.fasta.gz}. \\
+	
+	\noindent Then download GB\_TE dataset by using code (perl GB\_TE\_efetch.pl), the code can be found at \href{<url>}(\url{https://github.com/luzengAdelaide/Biogo-document/tree/master/Codes}). \\
+	
+	\noindent Retrovirus datasets can be downloaded from \url{https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi?taxid=11632}.\\
+	
+	\noindent Here is an example for wu-blast running:
+	
+	\noindent \textcolor{red}{1) Identify potential protain sequences}
+	\begin{lstlisting}[language=scala]
+	gzip -d uniprot_sprot.fasta.gz
+	
+	xdformat -p -k uniprot_sprot.fasta
+	
+	blastx ./report_run/sprot notKnown.fa  -gspmax=1 -E 0.00001 -B 1 -V 1 -cpus=32 > notKnown.fa.spwb
+	
+	python ./report_run/wublastx2gff.py notKnown.fa.spwb > notKnown.fa.spwb.gff
+	
+	\end{lstlisting}
+	
+	\noindent \textcolor{red}{2) Identify GB\_TE sequences}
+	\begin{lstlisting}[language=scala]
+	xdformat -p -k GB_TE.21032016.fa -o GB_TE.new
+	
+	blastx ./BlastDB/GB_TE.new notKnown.fa -gspmax=1 -E 0.00001 -B 1 -V 1 -cpus=32 > notKnown.fa.tewb
+	
+	python ./report_run/wublastx2gff.py notKnown.fa.tewb > notKnown.fa.tewb.gff
+	\end{lstlisting}
+	
+	\noindent \textcolor{red}{3) Identify potential retrovirus sequences}
+	\begin{lstlisting}[language=scala]
+	xdformat -n -k all_retrovirus.fasta
+	
+	tblastx ./BlastDB/all_retrovirus.fasta notKnown.fa -gspmax=1 -E 0.00001 -B 1 -V 1 -cpus=32 > notKnown.fa.ervwb
+	
+	python ./report_run/wublastx2gff.py notKnown.fa.ervwb > notKnown.fa.ervwb.gff
+	\end{lstlisting}
+	
+	\noindent \textbf{-gspmax:} max. number of gapped HSPs (GSPs) saved per subject sequence (default 0; 0 => unlimited). \\
+	\textbf{-B -V:} the B and V options limit the number of subject sequences for which any results whatsoever are reported, regardless of the number of HSPs or GSPs found. \\
+	\textbf{-E:} Expectation value (E) threshold for saving hits. \\
+	\textbf{-cpus:} no. of processors to utilize on multi-processor systems. \\
+	
+	\pagebreak
+	
+	\noindent Again, in this section, if you cannot/will not use wu-blast, ncbi-blast can be used instead, we tested different ncbi-blast parameters, to make the results are most consistent to the wu-blast results. \\
```

Comment:
The use of "will not" is inflammatory in English, just leave it is cannot.

Reply:
Done